INN/umbrella-mwcir
# Article title overlay on image

## Changes

This pull request makes the following changes:

- Adds styling that allows article title to be overlayed on the featured image.
- Adds secondary subtitle functionality

**Screenshots with title and image:**

**Full-width template:**

Desktop:
![image-full-width-desktop](https://user-images.githubusercontent.com/18353636/79141846-965e2d00-7d88-11ea-80d2-bf15f131fcc9.png)

Mobile: 
![image-full-width-mobile](https://user-images.githubusercontent.com/18353636/79141842-93633c80-7d88-11ea-95e1-46be264474a1.png)

**One column template:**

Desktop:
![image-one-column-desktop](https://user-images.githubusercontent.com/18353636/79141849-9827f080-7d88-11ea-8292-aa9e84aef095.png)

Mobile:
![image-one-column-mobile](https://user-images.githubusercontent.com/18353636/79141848-978f5a00-7d88-11ea-8962-68a1df5f7171.png)

**Two column template:**

Desktop:
![image-two-column-desktop](https://user-images.githubusercontent.com/18353636/79141854-98c08700-7d88-11ea-827a-4be807e937f2.png)

Mobile:
![image-two-column-mobile](https://user-images.githubusercontent.com/18353636/79141852-98c08700-7d88-11ea-98cf-e266498e5bdf.png)

**Screenshots without image, only title:**

**Full-width template:**

Desktop:
![no-image-full-width-desktop](https://user-images.githubusercontent.com/18353636/79142021-e0471300-7d88-11ea-84dd-53744b94a44c.png)

Mobile:
![no-image-full-width-mobile](https://user-images.githubusercontent.com/18353636/79142014-dde4b900-7d88-11ea-9788-48b1baa6835e.png)

**Two column template:**

Desktop:
![no-image-two-column-desktop](https://user-images.githubusercontent.com/18353636/79142024-e1784000-7d88-11ea-85d8-cdfa8072a0b7.png)

Mobile:
![no-image-two-column-mobile](https://user-images.githubusercontent.com/18353636/79142023-e0dfa980-7d88-11ea-8e8b-86f5adc5b1c3.png)

**One column template:**

Desktop:
![no-image-one-column-desktop](https://user-images.githubusercontent.com/18353636/79142026-e210d680-7d88-11ea-9b64-86079090cc20.png)

Mobile:
![no-image-one-column-mobile](https://user-images.githubusercontent.com/18353636/79142025-e210d680-7d88-11ea-8d9d-04d7c273a7aa.png)

## Why



For #39, #38

## Testing/Questions

Features that this PR affects:

- Article page titles, bylines, social icons in header, and top tags


Questions that need to be answered before merging:

- [x] Are there any templates I missed?
- [x] Any missing edge cases?

Steps to test this PR:

1. Find an article with a featured image and test it with all 3 template types (full, one col, two col) on desktop and mobile
2. Find an article with no featured image and test it with all 3 template types (full, one col, two col) on desktop and mobile
3. Add a subtitle to a post and make sure it displays as expected.
